InfiniteRecharge/src/main/java/frc/robot/operatorinterface/OI.java

```diff
@@ -85,6 +86,10 @@ public double manualAzimuthAxis() {
         return operatorControl.getRawAxis(OPERATOR_MANUAL_AZIMUTH_AXIS);
     }
 
+    public boolean feeder() {
+        return operatorControl.getPOV() == OPERATOR_FEEDER;
```

Comment:
This should be get raw button, not using the POV hat

Reply:
Oh thanks I wasn't thinking much last night, I was incredibly tired.

Reply:
Done.